azure-pipelines.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       inputs:
         toolMajorVersion: V2
         suppressionsFile: './scripts/ci/credscan/CredScanSuppressions.json'
+        toolVersionV2: '2.1.17'
```

Comment:
Doc for `toolVersionV2`: https://www.1eswiki.com/wiki/CredScan_Azure_DevOps_Build_Task